examples/trt-external-payload-cluster-density.yaml

```diff
@@ -35,6 +35,28 @@ tests :
         - "[Jira: kube-apiserver]"
       direction: 1
 
+    - name: multusCPU
```

Comment:
We should decide whether we want all metrics (including those not added here) to aggregate only the workload or also include the garbage collection metrics (lines 21-22 above).